src/test/java/com/exactpro/epfast/decoder/message/TestTemplatesWithGroup.kt

```diff
@@ -221,15 +221,15 @@ class TestTemplatesWithGroup {
     }
 
     companion object {
-        const val bytesString = BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 +
-                BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 + BufferStrings.ASCII_ZERO +
-                BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 + BufferStrings.ASCII_ZERO +
-                BufferStrings.ASCII_ABC + BufferStrings.ASCII_ABC + BufferStrings.ASCII_ZERO + BufferStrings.ASCII_ABC
-
-        const val nestedGroupBytesString = BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 +
-                BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 + BufferStrings.ASCII_ZERO +
-                BufferStrings.MANDATORY_INT32_942755 + BufferStrings.OPTIONAL_INT32_0 + BufferStrings.MANDATORY_INT32_942755 +
-                BufferStrings.OPTIONAL_INT32_0 + BufferStrings.ASCII_ZERO + BufferStrings.ASCII_ABC + BufferStrings.ASCII_ZERO +
-                BufferStrings.ASCII_ABC + BufferStrings.ASCII_ABC + BufferStrings.ASCII_ZERO + BufferStrings.ASCII_ABC
+        const val bytesString = FastSnippets.MANDATORY_INT32_942755 + FastSnippets.OPTIONAL_INT32_0 +
+                FastSnippets.MANDATORY_INT32_942755 + FastSnippets.OPTIONAL_INT32_0 + FastSnippets.ASCII_ZERO +
```

Comment:
I think based on the Kotlin coding style indent should be 4 spaces in this case. There is an option in IDEA to apply Kotlin coding style instead of outdated one. Ana knows where it is.